api4/oauth_test.go

```diff
@@ -154,6 +154,27 @@ func TestUpdateOAuthApp(t *testing.T) {
 	oapp.Id = "junk"
 	_, resp = AdminClient.UpdateOAuthApp(oapp)
 	CheckBadRequestStatus(t, resp)
+
+	th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableOAuthServiceProvider = true })
+	th.AddPermissionToRole(model.PERMISSION_MANAGE_OAUTH.Id, model.SYSTEM_USER_ROLE_ID)
+	th.LoginBasic()
+
+	userOapp := &model.OAuthApp{
```

Comment:
s/userOapp/oapp to maintain consistency with the rest of the test function.

Reply:
I called it differently because it's the only case where I create one as a user and not as a sysadmin.